app-emulation/virtiofsd/virtiofsd-1.5.1-r1.ebuild

```diff
@@ -128,6 +128,9 @@ src_unpack() {
 src_install() {
 	cargo_src_install
 
+	mkdir "${ED}/usr/libexec"
```

Comment:
|| die on both please (external commands which may fail).

Reply:
As for whether this is the right way to handle Rust: yes, usually it is :(

Cargo barely has install functionality and doesn't really allow much customisation at all. And unfortunately meson's Rust support isn't yet perfect when crates are involved (they're working on it).

So, this seems expected/OK to me.